src/Opdex.Platform.Infrastructure/Data/Handlers/Tokens/Snapshots/SelectTokenSnapshotsWithFilterQueryHandler.cs

```diff
@@ -116,7 +117,7 @@ private static string QueryBuilder(SelectTokenSnapshotsWithFilterQuery request)
         }
 
         // consider better solution, likely need to reuse
-        private SnapshotType ConvertToSnapshotType(Interval interval)
+        private static SnapshotType ConvertToSnapshotType(Interval interval)
```

Comment:
nitpic: Something I avoided initially as wasn't sure if it would be reused. Could be worth creating a static method in a separate class so we're not duplicating this everywhere.

Reply:
I see how it is, put it on the next guy huh haha